app/controllers/habits_controller.rb

```diff
@@ -0,0 +1,52 @@
+class HabitsController < ApplicationController
+  before_action :set_habit, only: [:show, :update]
+  before_action :set_user, only: [:index, :create]
+
+  def index
+    if @user
+      render json: @user.habits, status: 201
```

Comment:
## Status Code

Everything looks great, good work guys!  In general, the only thing I would change is the status codes of the index and show actions in the HabitsController.  From what I've been reading, status 201 is the code attributed to the successful creation of a record vs. 200 which is a successful GET request.  Also, 422 is the client-side error equivalent.  My first instinct is to label the show error as a 400, but that may be incorrect.

https://httpstatuses.com/
@markfranciose @sanjaynelson @tony-rodriguez